### PR TITLE
feat(EditableText): add isEditMode prop

### PR DESCRIPTION
**Pull request by talkor**
Due to requests, to make it controlled, approved by designers
https://monday.monday.com/boards/3532714909/pulses/5804938292

### CODE REVIEW

File: src/components/EditableTypography/EditableTypography.tsx [hunk 1]
```diff
@@ -24,6 +24,8 @@ export interface EditableTypographyImplementationProps {
   placeholder?: string;
   /** ARIA Label */
   ariaLabel?: string;
+  /** Controles the state of the component (i.e. view/edit mode) */
```
Comment (YossiSaadi) [inline src/components/EditableTypography/EditableTypography.tsx]: Controls*

File: src/components/EditableTypography/EditableTypography.tsx [hunk 2]
```diff
@@ -155,7 +165,10 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
           <AnimatedTypography
             ref={typographyRef}
             aria-hidden={isEditing}
-            className={cx(styles.typography, typographyClassName, { [styles.hidden]: isEditing })}
+            className={cx(styles.typography, typographyClassName, {
+              [styles.hidden]: isEditing,
+              [styles.disabled]: readOnly
```
Comment (YossiSaadi) [inline src/components/EditableTypography/EditableTypography.tsx:170]: was that a missing class? not related to the current feature?

**Review by YossiSaadi — APPROVED** [review]

File: src/components/EditableTypography/EditableTypography.tsx (hunk 2 above)
Comment (talkor) [reply src/components/EditableTypography/EditableTypography.tsx:170]: Yes, the class was there but the condition was missing

**Pull request merged**